pom.xml

```diff
@@ -587,7 +587,26 @@
       </plugins>
     </pluginManagement>
     <plugins>
-
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <version>2.3.0</version>
+        <configuration>
+          <javaVersion>8</javaVersion>
+          <ignoreClassNamePatterns>
+            .*.Input.*,.*Test,.*BadCommas
```

Comment:
Is there anyway to make it only exclude `Input` at the start of the file name?

Reply:
the first regex is for it

Reply:
The first regexp starts with `.*.` so I assume it will also match things like `ThisIsNotAnInput.java`.
It should only match things like `InputThisIs`

Reply:
@strkkk , please share details on what forcing you to do this regexp

Reply:
There is no good way to exclude folder (only packages/classes), so I wanted to exclude some test classes.
But I ended up with flag to exclude all test sources, because of what I wrote above and the fact that it was failing with some of the input
https://github.com/checkstyle/checkstyle/pull/10762#issuecomment-911645279

Reply:
So you get such failure if you switch `.*.Input.*` to `Input.*`?

Reply:
@rnveach there was some input files that were causing failures, but I couldn't find what those files are.

Original idea was to ignore inputs because some old things can be used there intentionally.

Reply:
Lines were removed, so I guess the issue is resolved.